Vagrantfile

```diff
@@ -226,7 +226,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       libvirt.memory      = vagrant_openshift_config['memory'].to_i
       libvirt.cpus        = vagrant_openshift_config['cpus'].to_i
       # run on libvirt somewhere other than default:
-      libvirt.uri         = ENV["VAGRANT_LIBVIRT_URI"] if defined? ENV["VAGRANT_LIBVIRT_URI"] 
+      libvirt.uri         = ENV["VAGRANT_LIBVIRT_URI"] unless defined? ENV["VAGRANT_LIBVIRT_URI"]
```

Comment:
This seems to be the opposite of what we want: set it only when it's not defined?

I had noticed it wasn't working too but hadn't created a PR with my fix. I think this may have been a recent change in vagrant-libvirt on fedora 22? Not sure, I thought it worked before as it was.

My change was: 

    libvirt.uri         = ENV["VAGRANT_LIBVIRT_URI"] || 'qemu:///system'

Can you try that out?